build/SignConfig.xml

```diff
@@ -3,12 +3,12 @@
   <job platform="" configuration="" dest="__INPATHROOT__" jobname="dep.controls" approvers="">
     <!-- Overwrite the input files to simplify downstream things like nuget and framework package generation. -->
     <file src="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
-    <file src="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\sdk\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\sdk\Microsoft.ProjectReunion.winmd" />
+    <file src="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" signType="WindowsRT" dest="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd" />
```

Comment:
>Microsoft.ProjectReunion.winmd [](start = 67, length = 30)

is winmd file architecture neutral?

Reply:
yes, typically people just pick the x86 version.  WinUI2 did some weird thing where they copied stuff from 'root' to sdk directory to re-write something.  We are not doing that.